src/bin/program.js

```diff
@@ -1,36 +1,45 @@
+'use strict';
+
 const program = require('commander')
 const { version } = require('../../package.json')
 const registerErrorHandler = require('./errors')
 const Logger = require('zos-lib').Logger;
+const chalk = require('chalk')
 
 const init = require('../commands/init')
-const addImplementation = require('../commands/add-implementation')
+const add = require('../commands/add-implementation')
 const push = require('../commands/push')
-const createProxy = require('../commands/create-proxy')
-const bumpVersion = require('../commands/bump-version')
-const upgradeProxy = require('../commands/upgrade-proxy')
-const linkStdlib = require('../commands/link-stdlib')
-const freeze = require('../commands/freeze')
+const create = require('../commands/create-proxy')
+const bump = require('../commands/bump-version')
+const upgrade = require('../commands/upgrade-proxy')
+const link = require('../commands/link-stdlib')
 const status = require('../commands/status')
+const freeze = require('../commands/freeze')
 
 program
   .name('zos')
   .usage('<command> [options]')
+  .description('where <command> is one of:\n' +
+          '\t add, bump, create, init, link, push, status, upgrade.')
   .version(version, '--version')
-  .option('-v, --verbose', 'Verbose mode. Output errors stacktrace and detailed log.')
-  .option('-s, --silent', 'Silent mode. Do not output anything to stderr.')
+  .option('-v, --verbose', 'verbose mode on: output errors stacktrace and detailed log.')
+  .option('-s, --silent', 'silent mode: no output sent to stderr.')
   .on('option:verbose', () => { Logger.verbose(true); } )
   .on('option:silent', () => { Logger.silent(true); } )
 
-init(program)
-addImplementation(program)
-push(program)
-createProxy(program)
-bumpVersion(program)
-upgradeProxy(program)
-linkStdlib(program)
-freeze(program)
-status(program)
+const commands = [init, add, push, create, bump, upgrade, link, status, freeze]
+
+commands.forEach((c) => {
+  c.setup(program)
+});
+
+const maxSig = Math.max(...commands.map(c => c.signature.length))
+program.on('--help', function(){
+  commands.forEach((c) => {
+    console.log('    '+chalk.bold(c.signature.padEnd(maxSig))+'\t'+c.description)
```

Comment:
I prefer interpolation with backticks, I find it more clear than concatenation; but it's a matter of personal preference